pkg/manager/internal/controllers/organizationroletemplate_controller.go

```diff
@@ -37,6 +44,10 @@ type OrganizationRoleTemplateReconciler struct {
 
 // +kubebuilder:rbac:groups=bulward.io,resources=organizationroletemplates,verbs=get;list;watch;update
 // +kubebuilder:rbac:groups=bulward.io,resources=organizationroletemplates/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=bulward.io,resources=organizations,verbs=get;list;watch;update
+// +kubebuilder:rbac:groups=apiserver.bulward.io,resources=projects,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Why are you accessing external, API extension proxy from within internal controller? Shouldn't you use internal project, not the filtered external one?

Reply:
This is needed actually. The reason are as following:
1. OrganizationRoleTemplate controller will create role/rolebinding for Organization Owners to provider permission for Owners to interact with `Projects`. I think User in Bulward should always interact with `Projects` in apiserver.bulward.io, not our core/internal group.
2. Then service account of controller-manager should have permission to create these roles.
That's why we need to guarantee it to create roles to interact with `Project` in apiserver API group, otherwise, we will need `escalate` verb for role, which I think is not nice :)

Reply:
got it!

Reply:
Maybe add a comment about it; that this is required due to privilege bestowing on the users.

Reply:
done